x-pack/plugins/alerting/server/lib/snooze/is_snooze_active.test.ts

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+import { RRule } from 'rrule';
+import sinon from 'sinon';
+import { RRuleRecord } from '../../types';
+import { isSnoozeActive } from './is_snooze_active';
+
+let fakeTimer: sinon.SinonFakeTimers;
+
+describe('isSnoozeExpired', () => {
+  afterAll(() => fakeTimer.restore());
+
+  test('snooze is NOT active byweekday', () => {
+    // Set the current time as:
+    //   - Feb 27 2023 08:15:00 GMT+0000 - Monday
+    fakeTimer = sinon.useFakeTimers(new Date('2023-02-27T08:15:00.000Z'));
+
+    // Try to get snooze end time with:
+    //   - Start date of: Feb 24 2023 23:00:00 GMT+0000 - Friday
+    //   - End date of: Feb 27 2023 06:00:00 GMT+0000 - Monday
```

Comment:
If you play with the `fakeTimer` you'll notice this snooze actually ends at `Feb 27 2023 07:00:00 GMT+0000`

Reply:
Sorry could you clarify this with an example? I think the snooze should always end at `Feb 27 2023 06:00:00 GMT+0000`. I tried a few different jest tzs and a few different faketimer times, could it be because spain had their daylight savings around march?

Reply:
Looks like the issue with calculating `now` that you fixed resolved this